README.md

```diff
@@ -24,17 +24,17 @@ This repository contains definitions for the following CloudEvents:
-|Cloud Scheduler|[Proto](proto/google/events/cloud/scheduler/v1/data.proto) / [JSON](https://googleapis.github.io/google-cloudevents/jsonschema/google/events/cloud/scheduler/v1/SchedulerJobData.json)|<br>Data Type:<br>`google.events.cloud.scheduler.v1.SchedulerJobData`<br>CloudEvent Type(s):<br>`google.cloud.scheduler.job.v1.executed`|
-|Cloud Storage|[Proto](proto/google/events/cloud/storage/v1/data.proto) / [JSON](https://googleapis.github.io/google-cloudevents/jsonschema/google/events/cloud/storage/v1/StorageObjectData.json)|<br>Data Type:<br>`google.events.cloud.storage.v1.StorageObjectData`<br>CloudEvent Type(s):<br>`google.cloud.storage.object.v1.finalized`<br/>`google.cloud.storage.object.v1.archived`<br/>`google.cloud.storage.object.v1.deleted`<br/>`google.cloud.storage.object.v1.metadataUpdated`|
-|Google Analytics for Firebase|[Proto](proto/google/events/firebase/analytics/v1/data.proto) / [JSON](https://googleapis.github.io/google-cloudevents/jsonschema/google/events/firebase/analytics/v1/AnalyticsLogData.json)|<br>Data Type:<br>`google.events.firebase.analytics.v1.AnalyticsLogData`<br>CloudEvent Type(s):<br>`google.firebase.analytics.log.v1.written`|
-|Firebase Authentication|[Proto](proto/google/events/firebase/auth/v1/data.proto) / [JSON](https://googleapis.github.io/google-cloudevents/jsonschema/google/events/firebase/auth/v1/AuthEventData.json)|<br>Data Type:<br>`google.events.firebase.auth.v1.AuthEventData`<br>CloudEvent Type(s):<br>`google.firebase.auth.user.v1.created`<br/>`google.firebase.auth.user.v1.deleted`|
-|Firebase Realtime Database|[Proto](proto/google/events/firebase/database/v1/data.proto) / [JSON](https://googleapis.github.io/google-cloudevents/jsonschema/google/events/firebase/database/v1/ReferenceEventData.json)|<br>Data Type:<br>`google.events.firebase.database.v1.ReferenceEventData`<br>CloudEvent Type(s):<br>`google.firebase.database.ref.v1.created`<br/>`google.firebase.database.ref.v1.updated`<br/>`google.firebase.database.ref.v1.deleted`<br/>`google.firebase.database.ref.v1.written`|
-|Firebase Remote Config|[Proto](proto/google/events/firebase/remoteconfig/v1/data.proto) / [JSON](https://googleapis.github.io/google-cloudevents/jsonschema/google/events/firebase/remoteconfig/v1/RemoteConfigEventData.json)|<br>Data Type:<br>`google.events.firebase.remoteconfig.v1.RemoteConfigEventData`<br>CloudEvent Type(s):<br>`google.firebase.remoteconfig.remoteConfig.v1.updated`|
+|-|-|-|-|
+|Cloud Audit Logs|[Proto](proto/google/events/cloud/audit/v1/data.proto) / [JSON](https://googleapis.github.io/google-cloudevents/jsonschema/google/events/cloud/audit/v1/LogEntryData.json)|<br>Data Type:<br>`google.cloud.audit.v1.LogEntryData`<br>CloudEvent Type(s):<br>`google.cloud.audit.log.v1.written`|
```

Comment:
As noted below, this is only the data type within the JSON schema - it's not the message type as declared by protobuf.

Reply:
No, my point was that the lack of ".event" means that it isn't the fully-qualified message name. That's fixed in the catalog itself in the last commit, but the README needs to be regenerated as well.

Reply:
I re-ran the generator, which should have the `.event`.

The CI step would have detected this.